include/cutlass/gemm/collective/xe_mma.hpp

```diff
@@ -137,12 +137,31 @@ struct CollectiveMma<
   using traits_load_B = Copy_Traits<GmemTiledCopyB, StrideB>;
   using atom_load_B = Copy_Atom<traits_load_B, ElementB>;
```

Comment:
I think the changes from this file need to be copied over to `xe_mma_mixed_input.hpp`. I am getting local failure of `ninja test_unit_gemm_device`

Reply:
done